segmentation/utils/spatial_analysis.py

```diff
@@ -20,6 +20,11 @@
 # pheno = pd.read_csv("/Users/jaiveersingh/Downloads/CellType_GlobalSpatialEnrichment/cellpheno_numkey.csv")
 # dist_mat = np.asarray(pd.read_csv("/Users/jaiveersingh/Documents/MATLAB/distancesMat5.csv", header=None))
 
+# Erin's Data Inputs-Neighborhood
+# dist_matrix = np.asarray(pd.read_csv("/Users/jaiveersingh/Documents/MATLAB/distancesMat5.csv", header=None))
+# all_data = pd.read_csv("/Users/jaiveersingh/Desktop/granA_cellpheno_CS-asinh-norm_matlab_revised.csv")
+# pheno_key = pd.read_csv("/Users/jaiveersingh/Downloads/CreateNeighborMatrix/cellpheno_numkey.csv")
+
```

Comment:
Recommend removing these comments prior to merging with master unless it's absolutely necessary to keep them around for reference sake.

Reply:
Should be removed once comparing with old output is no longer needed